### PR TITLE
Add meeting minutes for 31 October 2020

### PR DESCRIPTION
**Pull request by yongping827**
Added meeting minutes for 31 October 2020

### CODE REVIEW

File: docs/minutes/20201031.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+*Foreword: In lieu of the late release of mock practical exam bug reports, the meeting was postponed to 31 October 2020.*
+
+Happy halloween! 🎃👻🎃👻🎃
+
+Mock Practical Exam
+- 16 total issues
+- Nothing too serious
+- Got a time traveller on our hands (#246)
```
Comment (ianyong) [inline docs/minutes/20201031.md:8]: Possible to link to the various issues?

File: docs/minutes/20201031.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+*Foreword: In lieu of the late release of mock practical exam bug reports, the meeting was postponed to 31 October 2020.*
+
+Happy halloween! 🎃👻🎃👻🎃
```
Comment (ianyong) [inline docs/minutes/20201031.md:3]: Would be good to use `:jack_o_lantern:` and `:ghost:` instead.

**Review by ianyong — CHANGES_REQUESTED** [review]

File: docs/minutes/20201031.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+*Foreword: In lieu of the late release of mock practical exam bug reports, the meeting was postponed to 31 October 2020.*
```
Comment (ianyong) [inline docs/minutes/20201031.md:1]: Minutes does not appear in the list because the front matter is missing.

**Review by ianyong — CHANGES_REQUESTED** [review]

File: docs/minutes/20201031.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+layout: minutes
+date: 2020-10-23
```
Comment (ianyong) [inline docs/minutes/20201031.md:3]:
```suggestion
date: 2020-10-31
```

Wrong date.

**Review by ianyong — CHANGES_REQUESTED** [review]

File: docs/minutes/20201031.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+*Foreword: In lieu of the late release of mock practical exam bug reports, the meeting was postponed to 31 October 2020.*
+
+Happy halloween! :jack_o_lantern: :ghost: :jack_o_lantern: :ghost: :jack_o_lantern:
+
+Mock Practical Exam
+- 16 total issues
+- Nothing too serious
+- Got a time traveller on our hands [#246](https://github.com/AY2021S1-CS2103T-W16-3/tp/issues/246])
```
Comment (ianyong) [inline docs/minutes/20201031.md:13]:
```suggestion
- Got a time traveller on our hands [#246](https://github.com/AY2021S1-CS2103T-W16-3/tp/issues/246)
```

Broken link.

**Review by ianyong — CHANGES_REQUESTED** [review]

**Review by ianyong — APPROVED** [review]
LGTM!

**Review by wltan — APPROVED** [review]
LGTM!

**Pull request merged**